pkg/daemon/rpm-ostree.go

```diff
@@ -366,6 +368,10 @@ func (r *RpmOstreeClient) ApplyLive() (err error) {
 	return
 }
 
+func Cat(stateroot, commit, path string) ([]byte, error) {
```

Comment:
We could add a `Deployment` struct and this as a method; e.g. generate those structs from the status output.

Reply:
If you did `booted.Cat(path)`, would it be clear whether that would cat what was originally booted vs live applied?

Reply:
Oh good question!  Offhand I'd say `booted.Cat(path)` should always operate on the booted commit.  Querying the live filesystem state inherently doesn't *need* a special API, one can just call `open(path)` right?